modules/auxiliary/scanner/http/frontpage_credential_dump.rb

```diff
@@ -0,0 +1,79 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'FrontPage .pwd File Credential Dump',
+      'Description'    => %q{
+          This module downloads and parses the '_vti_pvt/service.pwd', '_vti_pvt/administrators.pwd', and '_vti_pvt/authors.pwd' files
+       on a FrontPage server to find credentials.
+      },
+      'References'     =>
+        [
+          [ 'URL', 'http://sparty.secniche.org/' ]
+        ],
+      'Author'         =>
+        [
+          'Aditya K Sood @adityaksood - Sparty tool',
```

Comment:
Typically the attribution is specified in a comment following the author's name:

```ruby
        [
          'Aditya K Sood @adityaksood', # Sparty tool
          'averagesecurityguy' # Metasploit module
        ],
```